.github/ISSUE_TEMPLATE/scenario-fail.yaml

```
@@ -0,0 +1,65 @@
+name: Scenario fail
+description: Report failures in scenario tests
+title: '[SCENARIO-FAILED] {Scenario} - {Brief description}'
+body:
+  - type: textarea
+    attributes:
+      label: Test Information
+      description: "Write the failed scenario information."
+      value: |
+        | Scenario | Catalog |
```

This is a minor point. 
I think it will be easier to find the failing scenario if we give the test suite + scenario name, instead of test catalog + scenario